icu4c/source/i18n/unicode/measunit.h

```diff
@@ -72,159 +72,270 @@ enum UMeasureUnitComplexity {
 };
 
 /**
- * Enumeration for SI prefixes, such as "kilo".
+ * Enumeration for SI and binary prefixes, e.g. "kilo-", "nano-", "mebi".
```

Comment:
You have a `-` after kilo and nano but not mebi

Reply:
Done.